src/lua/init.c

```diff
@@ -839,6 +839,13 @@ tarantool_lua_init(const char *tarantool_bin, int argc, char **argv)
 	}
 	luaL_openlibs(L);
 
+	/*
+	 * Create a table for storing loaded built-in modules.
+	 * Similar to _LOADED (package.loaded).
+	 */
+	lua_createtable(L, 0, 128);
```

Comment:
Minor: Why 128? I would rather use "`sizeof(lua_modules)`" for this.

Reply:
`lua_modules` is mostly irrelevant here. There are modules written using C. There are box's modules.

I did some calculations and got a number around 90 or 100 for this table. So 128 looks good to avoid rehashing and to don't have reasons to touch this for a long time. I can set it to zero if it is important. I definitelly will not write anything more complex than a constant here.

Reply:
OK, I get it now, thanks. Anyway, I would like to hear @Buristan opinion here. The most bothering thing here is the raw constant, but I can try to live with it :)

Reply:
Changed to `lua_newtable(L)`. No magic constants anymore.

Reply:
Thanks!